rts/motoko-rts-tests/src/gc/heap.rs

```diff
@@ -570,7 +570,7 @@ fn create_static_heap(
     // root.
     let array_addr = u32::try_from(heap.as_ptr() as usize).unwrap();
     let mut offset = 0;
-    write_word(heap, offset, TAG_ARRAY);
+    write_word(heap, offset, TAG_ARRAY_M); // TODO: TBR
```

Comment:
Alternatively, we could also use `TAG_ARRAY_I` for the static roots. But it is quite uncritical as it is not scanned by graph stabilization.

Reply:
(won't fix, as discussed)